<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit1">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>411</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>5</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>180</y>
      <width>270</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Source Initializing</string>
    </property>
    <widget class="QPushButton" name="pushButtonCam">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>26</y>
       <width>81</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Init Cam</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCamNum">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>27</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>81</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Init File</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>250</y>
      <width>270</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Work Control</string>
    </property>
    <widget class="QPushButton" name="pushButtonStart">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>26</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonStop">
     <property name="geometry">
      <rect>
       <x>148</x>
       <y>26</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>180</y>
      <width>240</width>
      <height>44</height>
     </rect>
    </property>
    <property name="title">
     <string>Image Format</string>
    </property>
    <widget class="QRadioButton" name="rbGrayF">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>18</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Gray</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbHSVF">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>18</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>HSV</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="rbBGRF">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>18</y>
       <width>70</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>RGB</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>240</y>
      <width>240</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Enable Tracking</string>
    </property>
    <widget class="QCheckBox" name="cbTrackBB">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>24</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Track Object</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbTrackFrame">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Track Ground</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>537</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <action name="menuRGB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="menuHSV">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>HSV</string>
   </property>
  </action>
  <action name="menuGray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
